src/components/CollapsedOpeningTimes.js

```diff
@@ -74,7 +87,10 @@ export default class CollapsedOpeningTimes extends React.Component {
         ];
 
         let openingHours = _(this.props.object.openingTimes)
-            .sortBy(record => order.indexOf(record.day));
+            .sortBy(record =>
+                (order.indexOf(record.day) * 24) +
+                (parseInt(record.open))
```

Comment:
The idea is that if you're open more than once in a day, we want the opening times to show the morning entry before the afternoon one..